python/ray/compat.py

```diff
@@ -0,0 +1,23 @@
+import errno
+import socket
+import sys
+
+
+def patch_redis():
```

Comment:
If we need to do this, then there should be a detailed comment explaining the issue (or a link to the relevant github issue).